app/src/renderer/vuex/getters.js

```diff
@@ -5,7 +5,8 @@ export const connected = state => state.node.connected
 export const lastHeader = state => state.node.lastHeader
 
 export const delegates = state => state.delegates
-export const shoppingCart = state => state.shoppingCart.delegates
+export const shoppingCart = state => state.delegation.delegates
+export const committedDelegations = state => state.delegation.committedDelegates
```

Comment:
how does `committedDelegations` differ from `bondedAtoms`? also, should this be...
```
export const committedDelegations = state => state.delegation.committedDelegations

Reply:
`committedDelegations` are our delegations which have been finalized on chain, the other values are just our local values before we have submitted.

That's actually the right name, called it `...Delegates` to match the `delegates` array inside the module